packages/golden-layout/src/LayoutManager.ts

```diff
@@ -513,6 +513,7 @@ export class LayoutManager extends EventEmitter {
       };
     }
 
+    config.id = config.id ?? getUniqueId();
```

Comment:
Update `getUniqueId` function to use `shortid` to generate the ID (or `nanoid` even, as we're replacing `shortid` soon: https://github.com/deephaven/web-client-ui/issues/1922)